ui/app/src/Router.tsx

```diff
@@ -18,7 +18,9 @@ import { ErrorBoundary, ErrorAlert } from '@perses-dev/components';
 import HomeView from './views/home/HomeView';
 import MigrateView from './views/MigrateView';
 
+// TODO follow same naming convention for all Views? (aka have the "View" term be either always a prefix or always a suffix).
 // Other routes are lazy-loaded for code-splitting
+const AdminView = lazy(() => import('./views/admin/AdminView'));
```

Comment:
I think there is a hack here, I noticed that there is a ``ViewDashboard`` used by routes, and inside , it is using another one named ``DashboardView`` :smile:

Reply:
So I would cope with the others here ViewAdmin = AdminView

Reply:
In the future it would be interesting to differentiate these view by a word in the name. I think there is one for perses app with backend queries and one more generic that just take everything as props